src/main/java/Blue/StorageHandler.java

```diff
@@ -0,0 +1,89 @@
+package Blue;
```

Comment:
can considr moving it to separate package

Reply:
Have moved everything into separate packages as well.